pulsar-broker/src/test/java/org/apache/bookkeeper/mledger/impl/ConsumerUnsubscribeIntegrationTest.java

```diff
@@ -1,84 +1,89 @@
-import org.awaitility.Awaitility;
-import org.testng.annotations.AfterClass;
-import org.testng.annotations.BeforeClass;
-import org.testng.annotations.Test;
-
-@Slf4j
-@Test(groups = "broker-impl")
-public class ConsumerUnsubscribeIntegrationTest extends ProducerConsumerBase {
-
-    @BeforeClass(alwaysRun = true)
-    @Override
-    protected void setup() throws Exception {
-        super.internalSetup();
-        super.producerBaseSetup();
-    }
-
-    @AfterClass(alwaysRun = true)
-    @Override
-    protected void cleanup() throws Exception {
-        super.internalCleanup();
-    }
-
-    @Test
-    public void testUnSubscribeWhenCursorNotExists() throws Exception {
-        final String topic = BrokerTestUtil.newUniqueName("persistent://public/default/tp");
-        final String subscription = "s1";
-        admin.topics().createNonPartitionedTopic(topic);
-        admin.topics().createSubscription(topic, subscription, MessageId.earliest);
-
-        Producer<String> producer = pulsarClient.newProducer(Schema.STRING).topic(topic).create();
-        Consumer<String> consumer = pulsarClient.newConsumer(Schema.STRING).topic(topic).subscriptionName(subscription)
-                .subscribe();
-
-        producer.send("1");
-        consumer.acknowledge(consumer.receive(2, TimeUnit.SECONDS));
-
-        PersistentTopic persistentTopic =
-                (PersistentTopic) pulsar.getBrokerService().getTopic(topic, false).join().get();
-        ManagedLedgerImpl ml = (ManagedLedgerImpl) persistentTopic.getManagedLedger();
-        ManagedCursorImpl cursor = (ManagedCursorImpl) ml.getCursors().get(subscription);
-        Awaitility.await().untilAsserted(() -> {
-            assertEquals(ManagedCursorImpl.State.Open, cursor.state);
-        });
-
-        // Verify: unsubscribe will succeed if the cursor was already removed.
-        ml.deleteCursor(subscription);
-        consumer.unsubscribe();
-
-        // cleanup
-        producer.close();
-        consumer.close();
-        admin.topics().delete(topic, false);
-    }
-}
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.bookkeeper.mledger.impl;
+
+import static org.testng.Assert.assertEquals;
+

```

Comment:
This doesn't look write. The code style in Pulsar doesn't have a blank line between imports.